pyproject.toml

```
@@ -23,14 +23,15 @@ transformers = {extras = ["ja", "sentencepiece"], version = "^4.38.1"}
 datasets = ">=2.17"
 sentence-transformers = "^3.0.0"
 pytest = "7.1.3"
-torch = "2.0.0"  # this version is needed to avoid "libcu*" related errors
+torch = "^2.3"  # this version is needed to avoid "libcu*" related errors
```

`2.0.0` に固定する必要は無くなったので、このコメントも必要なさそうですね
```suggestion
torch = "^2.3"
```